src/components/Flashcard/CardContent.js

```diff
@@ -1,45 +1,77 @@
 import React, { useState, useEffect } from 'react';
 import styles from './Flashcard.module.css';
-import { FaQuestion, FaCommentDots, FaSave, FaCheck, FaTimes } from 'react-icons/fa';
 
-import { Button } from 'components';
+import {
+    Button,
+    CrossIcon,
+    CheckIcon,
+    SaveIcon,
+    AnswerIcon,
+    QuestionIcon,
+} from 'components';
 
-export function CardContent({ question, addToUserBase, known, unknown}) {
-    const [isKnownAdded, setIsKnownAdded] = useState('iconButton');
-    const [isUnknownAdded, setIsUnknownAdded] = useState('iconButton');
+export function CardContent({ question, addToUserBase, known, unknown }) {
+    const [isKnownAdded, setIsKnownAdded] = useState('#416071');
+    const [isUnknownAdded, setIsUnknownAdded] = useState('#416071');
 
     useEffect(() => {
         isClicked(known, setIsKnownAdded);
         isClicked(unknown, setIsUnknownAdded);
```

Comment:
I haven't noticed this before, but it can lead to infinite loop of changes, so it could crash the app.

Reply:
I know about that but it doesnt work if I add a argument to dzis useEffect and dont know why, any suggestions?

Reply:
I can try to refactor this later